api/filtering.js

```diff
@@ -0,0 +1,79 @@
+const eventTypes = require("./eventTypes");
+
+function formatEvents(events, eventFilter) {
```

Comment:
are we filtering or formatting here? If we are doing both then either rename this file or, separate the filtering from the formatting logic, maybe in another file